File: backend/back_job/src/main/java/com/example/back_job/dto/UserDTO.java
package com.example.back_job.dto;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class UserDTO {
	private int user_id;
	private String last_name;
	private String first_name;
	private String email;
	private String phone;
	private String password;

}

File: backend/DTO/UserDTO.cs
using System.Text.Json.Serialization;

namespace backend.DTO
{
    public class UserDTO
    {
        public int UserId { get; set; }
        public string? FirstName { get; set; }
        public string? LastName { get; set; }
        public string? Email { get; set; }
        public string? Phone { get; set; }
        public string? Description { get; set; }
        public string? City { get; set; }
        public DateTime? CreationDate { get; set; }

        [JsonIgnore]
        public List<ApplicationsDTO>? AppCollection { get; set; }
        public List<UserCompetenciesDTO>? UserCompetencies { get; set; }
        public List<FormationsDTO>? Formations { get; set; }
        public List<ExperiencesDTO>? Experiences { get; set; }
    }

}